SYCL/FilterSelector/select_device_cpu.cpp

```diff
@@ -45,12 +45,19 @@ int main() {
     device d = cs.select_device();
     std::cout << "CPU device is found: " << d.is_cpu() << std::endl;
   }
-  // HOST device is always available regardless of SYCL_DEVICE_FILTER
+  /*
+  // TODO: enable this once SYCL_DEVICE_FILTER PR is merged.
```

Comment:
https://github.com/intel/llvm/pull/3397 needs to be merged first to enable this test.